medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/CasesController.cs

```diff
@@ -75,8 +76,8 @@ public async Task<ActionResult> SearchCaseByIdCode([Required][FromRoute] string
                 result = new PatientCase();
 
                 result.CaseId = @case.Item.CaseId;
-                result.DmerType = document.Item?.DmerType ?? string.Empty;
-                result.Status = document.Item?.Status ?? "Not Requested";
+                result.DmerType = string.IsNullOrEmpty(@case.Item?.DmerType) ? "Suspected Medical Condition" : @case.Item.DmerType;
```

Comment:
optional but this would be better mapped in cms-adapter. It makes sense to have these default values for any proto models leaving cms-adapter